CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Rojo Changelog
 
 ## Unreleased Changes
+* Skip sourcemap generation when unaffected by changes in watch mode ([#800])
```

Comment:
```suggestion
* Changed `sourcemap --watch` to only generate the sourcemap when it's necessary ([#800])
```
Changelogs are user facing, so I'd prefer it indicate what the change does rather than the precise nature of it internally.